lineman/app/components/navbar/navbar.haml

```diff
@@ -7,9 +7,9 @@
     .lmo-navbar__item{ng-class: '{"lmo-navbar__item--selected": selected == "inboxPage"}'}
       %a.btn.lmo-navbar__btn.lmo-navbar__btn-icon.has-badge{href: '/inbox', title: "{{ 'navbar.unread_title' | translate }}", tabindex: 2}
         %i.fa.fa-lg.fa-inbox>
+          %span.badge{ng-show: 'unreadThreadCount() > 0'}
```

Comment:
I don't think markup purity has anything to do with BEM, but yeah that
reasoning makes sense.

On Sat, Aug 8, 2015 at 2:19 AM, Aaron Thornton <notifications@github.com>
wrote:

> In lineman/app/components/navbar/navbar.haml
> <https://github.com/loomio/loomio/pull/2384#discussion_r36576676>:
>
> > @@ -7,9 +7,9 @@
> >      .lmo-navbar__item{ng-class: '{"lmo-navbar__item--selected": selected == "inboxPage"}'}
> >        %a.btn.lmo-navbar__btn.lmo-navbar__btn-icon.has-badge{href: '/inbox', title: "{{ 'navbar.unread_title' | translate }}", tabindex: 2}
> >          %i.fa.fa-lg.fa-inbox>
> > +          %span.badge{ng-show: 'unreadThreadCount() > 0'}
>
> I don't know if this has anything to do with font-awesome per se, seems
> pretty standard in iconography markup. The reason this is nested inside the *
> is that we want the badge to be positioned absolutely relative to the
> icon(which is fixed width). The button on the other hand is responsive and
> changes size based on the screen size. This is pretty common behaviour. If
> you were a markup purist you wouldn't be using BEM right?*
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2384/files#r36576676>.
>